singleheader/simdjson.cpp

```diff
@@ -342,7 +356,7 @@ static const uint64_t thintable_epi8[256] = {
 #ifndef SIMDJSON_HASWELL_IMPLEMENTATION_H
 #define SIMDJSON_HASWELL_IMPLEMENTATION_H
 
-/* isadetection.h already included: #include "isadetection.h" */
+/* isadetection.h already included: #include "isadetection.h"
 */
```

Comment:
It seems that your amalgamated version has those ^M chars in there. If you amalgamate on windows, you should force all files to use unit style line endings, otherwise you'll be getting all these spurious changes

Reply:
Also, irrelevant to the PR, these "already included:" comments are worthless. No c++/c compiler ever warns that some header was skipped because of headers guards and because it was already included. These should be removed imo

Reply:
Truth. And thanks, I'll make sure and reamalgamate on Linux.